### PR TITLE
Add instructions on copying and aliasing pages to the editor's guide

### LINKED ISSUE #7010: Add instructions on copying and aliasing pages to the Editor's guide
Currently there's no information on how to copy/alias a page in Wagtail. This feature is very nice, but hard to discover. Editors could benefit from some pointers and screenshots in the Editor's guide section of the Wagtail docs.

### PR DESCRIPTION
**Pull request by vladutmd**
Resolves #7010 

With this PR, adding a new page to the Editor's guide in the documentation regarding how to copy and create aliases to existing pages. I tried to follow the style of writing from other pages, let me know if there's anything I should change and/or add.

### CODE REVIEW

**Comment by zerolab** [comment]
Rendered: https://wagtail--7068.org.readthedocs.build/en/7068/editor_manual/copying_aliasing_existing_pages.html

File: docs/editor_manual/copying_aliasing_existing_pages.rst [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+
+* You are then taken to a new page where you can input the new information including the new title, new slug and choosing the parent page (if you want to change it). Please note that you can also publish the copied page right away and crate an alias (please see :ref:`aliasing-page-label` for more information about aliasing pages).
+
+
+.. image:: ../_static/images/screen12.6_2_copy_input_information.png
+   :alt: Copy page form with the options to change the title, slug, parent page, published status and option to create an alias.
+
+
+* Once you change the information required, press *Copy this page* and the page will be successfully copied and appear in the Explorer page.
```
Comment (jaap3) [inline docs/editor_manual/copying_aliasing_existing_pages.rst:26]:
```suggestion
* Once you completed this form, press *Copy this page*. The page will then be copied and appear in the Explorer page.
```

File: docs/editor_manual/copying_aliasing_existing_pages.rst [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+
+.. image:: ../_static/images/screen12.6_3_copy_success.png
+   :alt: Successful copying of the page resulting in it being available in an explorer page.
```
Comment (jaap3) [inline docs/editor_manual/copying_aliasing_existing_pages.rst:30]:
```suggestion
   :alt: Successful copying of a page results in it being available in the Explorer page.
```

File: docs/editor_manual/copying_aliasing_existing_pages.rst [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+
+
+.. _aliasing-page-label:
+
+Aliasing pages
+______________
+
+When copying a page, you also have the option to create an alias to it. Let's say you have a page about Brioche in the Breads section. At the same time, you might also want to have it in the Pastries section. You could create a copy and change both whenever there is a modification to be made. A more efficient way that ensures synchronisation between the two is aliasing the page during the copy process.
```
Comment (jaap3) [inline docs/editor_manual/copying_aliasing_existing_pages.rst:38]:
```suggestion
When copying a page, you also have the option to mark it as an alias. The content of an aliased page will stay in sync with the original. This is particularly useful when a page is required to available in multiple places.

For example, let's say you have a page about Brioche in the Breads section. You then want to also make this page available in the Pastries section. One way to do this is to create a copy of the Brioche page and change the parent page to the Pastries page. However you now need to remember to update this copy each time the original is modified. If a copy is marked as an alias Wagtail will do this for you each time a modification to the original page is published.
```

File: docs/editor_manual/copying_aliasing_existing_pages.rst [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+
+Here is how you can create an alias to an existing page:
```
Comment (jaap3) [inline docs/editor_manual/copying_aliasing_existing_pages.rst:40]:
```suggestion
Here is how to create an alias to an existing page:
```

File: docs/editor_manual/copying_aliasing_existing_pages.rst [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+
+* The first step is the same as it was for :ref:`copying-page-label`. When you hover over a page in an :doc:`Explorer page <finding_your_way_around/the_explorer_page>`, click on *More* and then *Copy*.
```
Comment (jaap3) [inline docs/editor_manual/copying_aliasing_existing_pages.rst:42]:
```suggestion
* The first step is the same as it is for :ref:`copying-page-label`. When you hover over a page in the :doc:`Explorer page <finding_your_way_around/the_explorer_page>`, click on *More* and then *Copy*.
```

File: docs/editor_manual/copying_aliasing_existing_pages.rst [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+* When you get to the copy page form, you can then choose another page as the parent page. Click the *Choose another page* button.
+
+
+.. image:: ../_static/images/screen12.7_1_alias_choose_parent_page_button.png
+   :alt: Clicking the choose another page button during the copy page form in orderto change the parent of the copied page.
```
Comment (jaap3) [inline docs/editor_manual/copying_aliasing_existing_pages.rst:47]:
```suggestion
   :alt: Clicking the choose another page button during the copy page form in order to change the parent of the copied page.
```

File: docs/editor_manual/copying_aliasing_existing_pages.rst [hunk 7]
```diff
@@ -0,0 +1,76 @@
...
+
+
+* Select and click on the new page to act as the parent page.
```
Comment (jaap3) [inline docs/editor_manual/copying_aliasing_existing_pages.rst:50]:
```suggestion
* Select and click on the desired parent page.
```

File: docs/editor_manual/copying_aliasing_existing_pages.rst [hunk 8]
```diff
@@ -0,0 +1,76 @@
...
+
+
+.. image:: ../_static/images/screen12.7_2_alias_choose_new_parent_page.png
+   :alt: Selecting and clicking on a new page to act as the new parent page.
```
Comment (jaap3) [inline docs/editor_manual/copying_aliasing_existing_pages.rst:54]:
```suggestion
   :alt: Selecting and clicking the desired parent page.
```

File: docs/editor_manual/copying_aliasing_existing_pages.rst [hunk 9]
```diff
@@ -0,0 +1,76 @@
...
+
+
+* Make sure the Alias checkbox is ticked. You can then complete the page aliasing by clicking on the *Copy this page* button.
```
Comment (jaap3) [inline docs/editor_manual/copying_aliasing_existing_pages.rst:57]:
```suggestion
* Make sure the *Alias* checkbox is ticked. 
* You can then complete the page aliasing by clicking on the *Copy this page* button.
```

File: docs/editor_manual/copying_aliasing_existing_pages.rst [hunk 10]
```diff
@@ -0,0 +1,76 @@
...
+
+
+.. image:: ../_static/images/screen12.7_3_alias_confirm_changes.png
+   :alt: Clicking on the Copy this page button to confirm aliasing.
+
+
+* The aliased page will then appear under the new parent page.
```
Comment (jaap3) [inline docs/editor_manual/copying_aliasing_existing_pages.rst:64]:
```suggestion
* The aliased page will then appear under the new parent in the Explorer.
```

File: docs/editor_manual/copying_aliasing_existing_pages.rst [hunk 11]
```diff
@@ -0,0 +1,76 @@
...
+
+
+.. image:: ../_static/images/screen12.7_3_alias_page_new_parent.png
+   :alt: The aliased page appearing under the new parent page.
```
Comment (jaap3) [inline docs/editor_manual/copying_aliasing_existing_pages.rst:68]:
```suggestion
   :alt: The aliased page appearing under the new parent page in the Explorer page.
```

File: docs/editor_manual/copying_aliasing_existing_pages.rst [hunk 12]
```diff
@@ -0,0 +1,76 @@
...
+
+
+* If you try to edit the aliased page, you will get notified that it is an alias of another page and be offered two options: either to edit the original page (with changes appearing in both places) or convert the alias page into a regular page (i.e. a regular copy).
```
Comment (jaap3) [inline docs/editor_manual/copying_aliasing_existing_pages.rst:71]:
```suggestion
* When you try to edit the aliased page, you are notified that it is an alias of another page. You will then be offered two options: edit the original page (with changes appearing in both places) or convert the alias page into a regular page (i.e. a regular copy, not updated when the original changes).
```

File: docs/editor_manual/copying_aliasing_existing_pages.rst [hunk 13]
```diff
@@ -0,0 +1,76 @@
...
+
+
+.. image:: ../_static/images/screen12.7_4_alias_page_edit_notification.png
+   :alt: Editing an alias page notifies it is an alias and offers to either edit the original page or change to a regular copy.
```
Comment (jaap3) [inline docs/editor_manual/copying_aliasing_existing_pages.rst:75]:
```suggestion
   :alt: Editing an alias page notifies it is an alias and offers to either edit the original page or change the alias to a regular page.
```

**Review by jaap3 — CHANGES_REQUESTED** [review]
Thank you so much for picking this up. I have some suggestions and fixed some typos. Feel free to ignore or partially apply some of the more extensive edits.

File: docs/editor_manual/copying_aliasing_existing_pages.rst (hunk 3 above)
Comment (vladutmd) [reply docs/editor_manual/copying_aliasing_existing_pages.rst:38]:
```suggestion
When copying a page, you also have the option to mark it as an alias. The content of an aliased page will stay in sync with the original. This is particularly useful when a page is required to be available in multiple places.
For example, let's say you have a page about Brioche in the Breads section. You then want to also make this page available in the Pastries section. One way to do this is to create a copy of the Brioche page and change the parent page to the Pastries page. However you now need to remember to update this copy each time the original is modified. If a copy is marked as an alias Wagtail will do this for you each time a modification to the original page is published.
```

**Comment by vladutmd** [comment]
Accepted the changes and fixed a typo, should be fine now :) thank you for the suggestions!

**Comment by vladutmd** [comment]
What is the test that failed?

**Comment by zerolab** [comment]
@vladutmd 

> docs/editor_manual/copying_aliasing_existing_pages.rst:58: D002 Trailing whitespace

**Comment by vladutmd** [comment]
Sorry, must have missed that. All set!

**Comment by vladutmd** [comment]
any updates on this making it in?

**Comment by zerolab** [comment]
Hey @vladutmd,

Will aim to get this reviewed in the next few days with a view to getting it in.
Thank you for you contribution and sorry for the delayed response!

**Comment by zerolab** [comment]
Merged in f54e7e1e416. Thank you very much for this @vladutmd

**Pull request closed without merging**